libs/librepcb/projecteditor/cmd/cmdmirrorselectedschematicitems.cpp

```diff
@@ -108,9 +108,12 @@ bool CmdMirrorSelectedSchematicItems::performExecute() {
     appendChild(cmd);
   }
   foreach (SI_NetLabel* netlabel, query->getNetLabels()) {
+    // Since there is no right alignment (yet), coordinates need to be re-adjusted to accomodate left shift
+    // New position = mirrored old position - label width
+    Point newpos = netlabel->getPosition().mirrored(mOrientation, center);
+    newpos.setX(newpos.getX() - netlabel->getWidth());
```

Comment:
As the width of the netlabel is not mapped to grid, this will make the netlabel be located off the grid after mirroring. I would propose to map the width to grid first to avoid off-the-grid positions, although then the mirror operation is less accurate:

```suggestion
    newpos.setX(newpos.getX() - netlabel->getWidth().mappedToGrid(mSchematic.getGridProperties().getInterval()));
```